lib/ansible/modules/network/iosxr/iosxr_command.py

```diff
@@ -174,6 +175,8 @@ def main():
 
     spec.update(iosxr_argument_spec)
 
+    spec.update(command_spec)
```

Comment:
Adding `command_spec`  as part of top-level spec is not the right way to use of `ComplexList

Reply:
As discussed, will keep it as is; for later modification in #33575